src/sentry/rules/processing/delayed_processing.py

```diff
@@ -241,9 +239,9 @@ def get_group_to_groupevent(
 def process_delayed_alert_conditions(buffer: RedisBuffer) -> None:
     with metrics.timer("delayed_processing.process_all_conditions.duration"):
         project_ids = buffer.get_set(PROJECT_ID_BUFFER_LIST_KEY)
-        for project_id in project_ids:
+        for project_id, date_added in project_ids:
```

Comment:
Redis responds with a set of tuples of the rule id and the timestamp of when it was added, this was incorrect before.